src/main/java/com/manywho/services/saml/adapters/ManyWhoSaml2Settings.java

```diff
@@ -57,6 +62,15 @@ public ManyWhoSaml2Settings(Configuration configuration) {
             throw new RuntimeException(e);
         }
 
+        if (!StringUtils.isEmpty(configuration.getSpPrivateKey())) {
+            try {
+                this.setSpPrivateKey(parsePrivateKey(configuration.getSpPrivateKey()));
+            } catch (Base64DecodingException | InvalidKeySpecException | NoSuchAlgorithmException e) {
+                e.printStackTrace();
+                throw new RuntimeException(e);
```

Comment:
@matt-manywho  Could you please change this to something like 
````
throw new RuntimeException("Error parsing Service Provider Private Key", e);
````
It will more easy to understand what is going on for the user if something goes wrong.